crypto/src/main/scala/org/bitcoins/crypto/ECDigitalSignature.scala

```diff
@@ -121,6 +121,16 @@ object ECDigitalSignature extends Factory[ECDigitalSignature] {
     }
   }
 
+  def fromFrontOfBytes(bytes: ByteVector): ECDigitalSignature = {
+    val sigWithExtra = fromBytes(bytes)
+    val sig = fromRS(sigWithExtra.r, sigWithExtra.s)
+
+    require(bytes.startsWith(sig.bytes),
+            s"Received weirdly encoded signature at beginning of $bytes")
+
```

Comment:
Should you require that `sig == sigWithExtra`?

Reply:
no, `sigWithExtra` has all the bytes